### ISSUE TITLE
Memoryerror with 384,000 sampling rate

### ISSUE THREAD

**Issue by pickingname** [OpenNBS/NoteBlockStudio#394]
**Describe the bug**

**To Reproduce**
1. click file
2. click export as audio track
3. change the sampling rate to 384,000
4. wait untill the error pops ups

**Expected behavior**
exports with 384,000 sampling rate

**Screenshots**
![image](https://github.com/OpenNBS/OpenNoteBlockStudio/assets/115550149/fe57d2e3-db57-47d1-85c1-a1e92b4d5b29)

**Additional context**
i dont know is it because it has been used all of my ram
windows 10, 24gb ram!!

Also i imported the song from midi

**Comment by Bentroen** [comment]
Hi! Could you please check (and if possible, attach a screenshot of) your RAM usage in the Task Manager right before the error appears? Since NBS is a 32-bit executable, I'd like to know if it's related to the 32-bit memory address limit (4 GB max) or if something else is causing the issue.

**Comment by Bentroen** [comment]
Sorry, I meant the memory usage of NBS (which you can see in the 'Processes' tab), not your entire system's RAM usage :)
Could you check whether it gets close to 4 GB as the error is about to occur? (Looking at the graph it doesn't seem like it, I'd just like to eliminate or confirm that this is causing the error).

> it looks like the app that is focused is issuing the error.

That's a different issue we can't do much about, as the emission of popups is handled by GameMaker (the engine that NBS developed on) and we've got little control over it.

**Comment by Bentroen** [comment]
Yeah, the memory usage spikes, but it doesn't seem to get close to 4 GB, so the problem must be caused by something else.

Would you mind attaching the affected song to this issue so it's easier to reproduce?
(You can also send it privately to me on Discord: Bentroen#7626)
